src/shared/services/filter.service.ts

```diff
@@ -5,20 +5,20 @@ import {
   ICalculatedProduct,
   IFilterOption,
   IFilterDefinition,
-} from '../models';
-import { ProductFilterOption } from '../enums';
-import { CategoryFacadeService, BrandFacadeService, FilterFacadeService } from '../services';
+  ProductFilterOption,
+  CategoryFacadeService,
+  BrandFacadeService,
+} from '@shared-module';
 import { TranslateService } from '@ngx-translate/core';
 import { firstValueFrom } from 'rxjs';
 import { Store } from '@ngrx/store';
-import { FilterActions } from '../store';
+import { FilterActions } from '../store/filters';
```

Comment:
Why did you change the imports here? It was correct as it was before, because we are using index files for importing.